akka-docs/src/main/paradox/scala/futures.md

```diff
@@ -123,23 +196,19 @@ each step of the for comprehension is run sequentially. This will happen on sepa
 there isn't much benefit over running the calculations all within a single `Future`.
 The real benefit comes when the `Future`s are created first, and then combining them together.
 
+@@@
+
 ### Composing Futures
 
+@@@ div { .group-scala }
+
 The example for comprehension above is an example of composing `Future`s.
 A common use case for this is combining the replies of several `Actor`s into a single calculation
 without resorting to calling `Await.result` or `Await.ready` to block for each result.
 First an example of using `Await.result`:
 
 @@snip [FutureDocSpec.scala]($code$/scala/docs/future/FutureDocSpec.scala) { #composing-wrong }
 
-@@@ warning
```

Comment:
This was moved to the line 67, where `Await` first appeared on this page, which is consistent with Java.